packages/serve/package.json

```diff
@@ -15,22 +15,14 @@
     "chalk": "2.4.2",
     "cross-spawn": "6.0.5",
     "inquirer": "6.5.1",
-    "webpack-dev-server": "^3.8.1"
+    "webpack-dev-server": "^3.9.0"
   },
   "devDependencies": {
     "@types/cross-spawn": "6.0.0",
     "@types/inquirer": "6.5.0",
     "@types/node": "12.7.2",
     "typescript": "3.5.3"
   },
-  "peerDependencies": {
```

Comment:
Could you set this to `>= 3` and optional to false? After that, we are good! 

CC @evilebottnawi